ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -280,9 +291,20 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1CourtOrderPayment",
+    "CaseFieldID": "respondent1CourtOrderPaymentLabel",
     "PageDisplayOrder": 18,
     "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageShowCondition": "specDefenceAdmittedRequired = \"No\" AND defenceAdmitPartPaymentTimeRouteRequired!=\"IMMEDIATELY\"",
```

Comment:
When I tried to run ccd import, there should be only one page show condition for this page but there is two here.

Reply:
I can only find one occurrence of "DetailsOfPayingMoneyRepaymentPlan". Where is the second page show condition?

Reply:
one of them will create a header and other one will display the rest of page, I had already explained that in show and tell.

Reply:
It is line in 309 and line 310
"PageShowCondition": "specDefenceAdmittedRequired = \"No\" AND defenceAdmitPartPaymentTimeRouteRequired=\"*\" AND defenceAdmitPartPaymentTimeRouteRequired!=\"IMMEDIATELY\"",
    "PageID": "DetailsOfPayingMoneyRepaymentPlan",

Reply:
Now I see it, lines 299 and 310. PageShowCondition is twice for the same PageID. Maybe one of them should be **Field**ShowCondition instead of PageShowCondition?

Reply:
dario, second condition is not required